app/javascript/app/components/my-climate-watch/my-visualisations/my-cw-vis-graph/my-cw-vis-graph-actions.js

```diff
@@ -0,0 +1,22 @@
+import { createAction, createThunkAction } from 'redux-tools';
+import { CWAPI } from 'services/api';
+
+export const fetchVisualisationsInit = createAction('fetchVisualisationsInit');
```

Comment:
There is a typo in this action. It should be `fetchVisualisationInit` (visualisation in singular). Webpack is sad because of it 😢